doc/guides/bluetooth/bluetooth-arch.rst

```diff
@@ -393,9 +393,9 @@ configuration options needed will look something like the following:
     CONFIG_FLASH=y
     CONFIG_FLASH_PAGE_LAYOUT=y
     CONFIG_FLASH_MAP=y
-    CONFIG_FCB=y
+    CONFIG_NVS=y
     CONFIG_SETTINGS=y
-    CONFIG_SETTINGS_FCB=y
+    CONFIG_SETTINGS_NVS=y
```

Comment:
CONFIG_SETTINGS_NVS is default if CONFIG_NVS, So I think we can remove this line.

Reply:
removed